os/manual-rollbacks.md

```diff
@@ -2,6 +2,8 @@
 
 As much as we all love flawless, automatic updates, there may be occasions when an update must be rolled back. This is fairly straightforward on Container Linux, once you know the right commands.
 
+**Note**: Container Linux allows you to rollback to a previously active partition, or change to an older release channel (to downgrade to a stable version). It does not allow you to downgrade to an uninstalled, user defined version number.
```

Comment:
If that's true, this whole page should be repositioned - and it's not just a matter of adding a note at the top. As it is, the page doesn't describe a 'boot failed, you must rollback' process. It seems to describe a rollback after a successful update. Please confirm, and I'll take a stab at it.

I will ask 1) whether the rollback happens automatically, and 2) if not, what's the error message presented when reboot fails?